keps/sig-api-machinery/1929-built-in-default/README.md

```diff
@@ -414,8 +416,14 @@ Both these cases will be handled properly, here's an example:
 type Object struct {
      // This field has an invalid zero-value, empty string is not a
      // valid string.
-     // +default=default-name
-     Name string
+     //
+     // If this is a CRD, there would be a kubebuilder specific tag, and
+     // if this is a built-in, there would be a specific validation function,
+     // but basically:
+     // +minLength=1
```

Comment:
does this mean `minLength: n` for some n>0 is hardcoded as the signal to switch to the new behaviour? If not, see comment above: it's getting complicated.

Reply:
Please add an example for `type: number`. How do you want to express `!= 0.0`? Same for integer.